model/config.go

```diff
@@ -3564,3 +3580,63 @@ func (o *Config) Sanitize() {
 		*o.ServiceSettings.GfycatApiSecret = FAKE_SETTING
 	}
 }
+
+// structToMap converts a struct into a map removing those fields that has the tag passed
+// as argument
+func (o *Config) structToMapFilteredByTag(t interface{}, typeOfTag, filterTag string) map[string]interface{} {
```

Comment:
Same as [here](https://github.com/mattermost/mattermost-server/blob/d535e666380b81876b4410d0837da76aa9b4f461/cmd/mattermost/commands/utils.go#L22) but with the tag filter logic added. I was thinking on adding it to some util.go file but this is specific for Config so I didn't see the benefit